spring-cloud-skipper-server/src/main/java/org/springframework/cloud/skipper/service/ReleaseService.java

```
@@ -178,6 +178,15 @@ public Release delete(String releaseName) {
 		return this.releaseManager.delete(release);
 	}
 
+	public Info status(String releaseName) {
+		Release release = this.releaseRepository.findTopByNameOrderByVersionDesc(releaseName);
+		if (release != null) {
+			release = status(release);
+			return release != null ? release.getInfo() : null;
+		}
+		return null;
```

throw ReleaseNotFoundException here instead of returning null.